### DISCUSSION TITLE
We need a promotional effort by the foundation with reputable gambling websites and in Decentraland.

### DISCUSSION THREAD

**Discussion by didlie** [dogecoin/dogecoin#2845] [General]
By targeting GG Poker, and other legal New Jersey and Las Vegas gambling websites to incorporate Dogecoin into their platforms, the utility of Dogecoin will be greatly improved, and the foundation can avoid the need to comply with gambling laws because the legal platforms are already compliant.

A presence and investment in Decentraland by the Dogecoin foundation will greatly enhance the visibility of Dogecoin.

**Comment by rnicoll** [comment]
I mean the position for Dogecoin historically has always been that it aims to avoid promoting such activities, at least directly (we know it happens, of course, but it's not how we present Dogecoin).

**Reply by didlie** [reply]
...the greatest utility for crypto is dark-web shopping (whatever that means). So, no value or respectability in the pretense of an ethical standard for a Dog-e coin above the world-wide respectability and popularity of professional and hobby poker and sports betting. Have you ever chatted with Daniel Negreanu?

**Reply by tealethetoolman** [reply]
The greatest utility of crypto is peer to peer commerce. Dark web and gambling are all negative publicity. Go tell your favorite stores and restaurants about bitpay so that people can use doge in your are. Good publicity is better than scams.

**Reply by didlie** [reply]
https://dogechain.info/block/4107515

Empty blocks... The network is not useful... Declining value .. and you think you are better than millions of poker players and sports betters... Find some use for the network, or watch it perish

**Comment by didlie** [comment]
Additionally, companies like Atari, Animoca Brands, and Delta Time should be targeted to incorporate Dogecoin into their crypto-rewards oriented gaming platforms. The symbiosis of Dogecoin (one of the most popular crypto currencies for traders) and these gaming platforms (that now rely on less interesting and less liquid tokens) will greatly enhance the visibility and desirability of both Dogecoin and their gaming and virtual products.

**Reply by didlie** [reply]
...feel free to move this comment into a new discussion thread.

**Comment by didlie** [comment]
Dogecoin is better than Bitcoin for tipping on Twitter. Network fees for small transactions on Bitcoin make it less useful than Dogecoin. Is anyone making an effort to integrate with Twitter, YouTube, Twitch etc ?

**Comment by didlie** [comment]
Small and fast growing gambling website companies, like Kings Entertainment Group Inc., need to be approached by the Dogecoin foundation for cooperative marketing efforts and Dogecoin integration into their platforms. Legal webservices are the best approach for expanding the viability and utility of the Dogecoin network.

**Reply by patricklodder** [reply]
Hi! This repository is not covering Dogecoin Foundation. Could you please take this to their repository at https://github.com/dogecoinfoundation or contact them on their discord?

Thanks
